images/macos/toolsets/toolset-15.json

```
@@ -31,17 +42,23 @@
                     "sha256": "4a26c3d102a55c7222fb145e0ee1503249c9c26c6e02dc64d783c8810b37b1e3",
                     "symlinks": ["16.0"],
                     "install_runtimes": "none"
-                },
-                {
-                    "link": "15.4",
-                    "version": "15.4.0+15F31d",
-                    "sha256": "82d3d61804ff3f4c7c82085e91dc701037ddaa770e542848b2477e22f4e8aa7a",
-                    "install_runtimes": "default"
                 }
             ]
         },
         "arm64":{
             "versions": [
+                {
+                    "link": "16.4_beta", 
+                    "version": "16.4-Beta+16F1t",
```

```suggestion
                    "link": "16.4_Release_Candidate", 
                    "version": "16.4_Release_Candidate+16F6",
```
Given RC1 for Xcode 16.4 is already out ?